app/src/main/res/values-pcm-rNG/strings.xml

```diff
@@ -523,15 +523,15 @@
   <string name="faq_answer_create_profile">&lt;p&gt;If na your first time creating a profile and not have a PIN:&lt;ol&gt;&lt;li&gt;From di Profile Chooser, tap on &lt;strong&gt;Set up Multiple Profiles&lt;/strong&gt;.&lt;/li&gt;&lt;li&gt;Create a PIN and &lt;strong&gt;Save&lt;/strong&gt;.&lt;/li&gt;&lt;li&gt;Fill in all boxes for di profile.&lt;ol&gt;&lt;li&gt;(Optional) Upload a photo.&lt;/li&gt;&lt;li&gt;Enter a name.&lt;/li&gt;&lt;li&gt;(Optional) Assign a 3-digit PIN.&lt;/li&gt;&lt;/ol&gt;&lt;/li&gt;&lt;li&gt;Tap &lt;strong&gt;Create&lt;/strong&gt;. Dis profile go add to your Profile Chooser!&lt;/li&gt;&lt;/ol&gt;&lt;/p&gt;&lt;p&gt; If you don create a profile before and you get a PIN:&lt;ol&gt;&lt;li&gt;From di Profile Chooser, tap on &lt;strong&gt;Add Profile&lt;/strong&gt;. &lt;/li&gt;&lt;li&gt;Enter your PIN and tap &lt;strong&gt;Submit&lt;/strong&gt;. &lt;/li&gt;&lt;li&gt;Fill in all boxes for di profile.&lt;ol&gt;&lt;li&gt; (Optional) Upload a photo. &lt;/li&gt;&lt;li&gt; Enter a name. &lt;/li&gt;&lt;li&gt; (Optional) Assign a 3-digit PIN. &lt;/li&gt;&lt;/ol&gt;&lt;/li&gt;&lt;li&gt;Tap &lt;strong&gt;Create&lt;/strong&gt;. Dis profile go add to your Profile Chooser!&lt;/li&gt;&lt;/ol&gt;&lt;/p&gt;&lt;br&gt;&lt;p&gt;Note: Only di &lt;u&gt;Administrator&lt;/u&gt; go dey able to manage profiles.&lt;/p&gt;</string>
-  <string name="faq_answer_app_language" fuzzy="true">&lt;p&gt;The %s app dey support English, Brazilian Portuguese, Arabic, Swahili, and Nigerian Pidgin. Select one of these languages for di menu, under Options. To ask for the app for your language, abeg contact us for <strong>admin@oppia.org</strong>.&lt;/p&gt;</string>
-  <string name="faq_answer_bug_reporting" fuzzy="true">&lt;p&gt;<ol><li>From your %s app home screen, tap the menu for di top left corner.</li><li>Tap <strong>Share feedback</strong>.</li><li>Follow di instructions to report di bug or share feedback.</li></ol>&lt;/p&gt;</string>
-  <string name="faq_answer_math_lessons" fuzzy="true"><p>%1$s mission na to help learners gain necessary life skills. Math na essential skill for everyday life. %1$s go dey offer new lessons on science and other subjects very soon!</p></string>
-  <string name="faq_answer_more_lessons" fuzzy="true"><p>Yes, %s go dey offer new lessons on science and other subjects very soon. Abeg check back for updates!</p></string>
+  <string name="faq_answer_app_language" fuzzy="true">&lt;p&gt;The %s app dey support English, Brazilian Portuguese, Arabic, Swahili, and Nigerian Pidgin. Select one of these languages for di menu, under Options. To ask for the app for your language, abeg contact us for .&lt;/p&gt;</string>
+  <string name="faq_answer_bug_reporting" fuzzy="true">&lt;p&gt;&lt;/p&gt;</string>
```

Comment:
This stringand the two below are broken too. In the immediate last force push, it was okay but now it's empty.